### DISCUSSION TITLE
any reports of bans?

### DISCUSSION THREAD

**Discussion by frosDfurret** [GiorgioBrux/nitro-sniper-enhanced#117] [Q&A]
just wondering since this isn't my first time using a sniper, and i'm scared my main account (the only one i'm using at the moment) is going to get banned like my 5 other alts did.
unsure of any nitro timeout or anything, so very scared that the sniper will snipe too many codes at a time and disable my main account.

**Comment by GiorgioBrux** [comment]
As long as you use only one instance and don't have 999999 years of nitro it's probably fine. Make sure you set `useMain` to false for additional security.
